mobile-first/index.html

```diff
@@ -0,0 +1,117 @@
+<!doctype html>
```

Comment:
You need not have another index file for the mobile first. Instead you style the mobile view first and start showing hidden elements as the screen gets wider.